src/components/ecosystems/CodeSplit/CodeSplit.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { PureComponent } from 'react';
+
+import { Bundle } from 'common/components';
+
+const loadMyDep = () =>
+    import(/* webpackChunkName: "myDependancy" */ './myDep.js');
+
+export default class CodeSplit extends PureComponent {
```

Comment:
Can't this just be a pure function vs a React component?

Reply:
Yes, it was and I changed it to test something. Updating